resources/js/app.ts

```diff
@@ -2,10 +2,17 @@ import Alpine from 'alpinejs';
 // eslint-disable-next-line camelcase,import/no-unresolved
 // import { livewire_hot_reload } from 'virtual:livewire-hot-reload';
 
+import { lazyLoadModuleOnIdFound } from './lazyLoadModuleOnIdFound';
 import { clipboard, mobileSafeTipEvents, themeChange } from './utils';
 
 // livewire_hot_reload();
 
+lazyLoadModuleOnIdFound({
+  elementId: 'reorder-site-awards-header',
+  codePath: './reorderSiteAwards',
+  moduleName: 'reorderSiteAwards',
+});
```

Comment:
I'm assuming this is failing for me. Any changes I attempt to make generate an error similar to this:
```
Uncaught ReferenceError: reorderSiteAwards is not defined
    at HTMLButtonElement.onclick
```

Reply:
This lazy loading functionality should now be working properly and respond to changes while in dev mode.